pc/manager.cpp

```diff
@@ -62,7 +62,8 @@ manager::manager()
   do_cap_( false ),
   do_tx_( true ),
   is_pub_( false ),
-  cmt_( commitment::e_confirmed )
+  cmt_( commitment::e_confirmed ),
+  sreq_{ { commitment::e_processed } }
```

Comment:
Why `processed` vs. `confirmed` here?

Reply:
`slotSubscribe` is giving `processed` rather than `confirmed`. So this is giving the same behavior. Open to changing to `confirmed`, but I feel that would be a separate change. In practice, I noticed these tend to be 1 apart.